config/default/shifts.json

```diff
@@ -0,0 +1,74 @@
+{
+    "R" : {
```

Comment:
Czy potrzebne są te klucze? Nie lepiej zrobić listę JSON objectów i w kodzie sobie to aggregować jak się chce?

Reply:
Też jest to opcja, mogę podejrzeć jak z implementacją tego, bo konwencję kluczy przyjąłem z faktu analogii do starego scoringu. Ewentualnie też mogę wywalić wymaganie tych kluczy w JSON'ie, żeby sobie sam generował słownik z tablicy.